zproject/backends.py

```diff
@@ -275,6 +320,20 @@ def ldap_to_django_username(self, username: str) -> str:
         return username
 
 class ZulipLDAPAuthBackend(ZulipLDAPAuthBackendBase):
+
+    def __init__(self) -> None:
+        mode = settings.FAKE_LDAP_MODE
+
+        if settings.DEVELOPMENT and mode:  # nocoverage # We only use this in development
```

Comment:
@timabbott I'm not sure about the `# nocoverage`, you might want to have a look at this.